.github/workflows/release.yaml

```diff
@@ -10,12 +10,12 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
       
       - name: Use Node.js
         uses: actions/setup-node@v3
```

Comment:
@BugGambit Small nit, but this also should be updated to v4

Reply:
4a04b9d209